src/effects/native/metronomeclick.h

```diff
@@ -0,0 +1,1034 @@
+#ifndef METRONOMECLICK_H
+#define METRONOMECLICK_H
+
+#include "util/types.h"
+
+const unsigned int kClickSize = 1024;
+const CSAMPLE kClick[kClickSize] = {
```

Comment:
How did you generate these values?

Reply:
This is a recording from my daughter's metronome.

Reply:
I have used Audacity to export the samples as text.

Reply:
Please add a comment noting this.